File: CppCompiler/nestedforloop10.cpp
#include<iostream>
using namespace std;

int main()
{
  double i,j,n,fact=1,sum=0,sum1=0,k;
cout<<"enter range";
cin>>n;
for(i=1;i<=n;i++)
{
for(j=1;j<=i;j++)
{
cout<<j<<"!";
for(k=1;k<=j;k++)
{
fact=fact*k;
}
sum1=sum1+fact;
fact=1;
if(j<i)
{
cout<<"+";
}
}
cout<<"/"<<i;
if(i<n)
{
cout<< " + ";
}
sum=sum+(double)sum1/i;
sum1=0;
}
cout<<"sum of seriese="<<sum;
}

File: python programming/nestedforloop10.py
n=int(input("enter range of pime::-"))
sum=0
k=1
for i in range(2,n+1):
	f=0
	for j in range(2,int(i/2)+1):
		if i%j==0:
			f=1
	if f==0:
		if i<=n and k!=1:
			print(' + ',end='')

		print(f'{k}/{i}',end='')
		sum+=k/i
		k+=1
		
print("\b\n sum of seriese=",sum)
	
		
		
		
		